dind/dind-up.sh

```diff
@@ -18,7 +18,8 @@ KUBE_ROOT=${KUBE_ROOT:-"../../kubernetes"}
 K8S_VERSION=$(cat ${KUBE_ROOT}/bazel-out/stable-status.txt | grep STABLE_DOCKER_TAG | awk '{print $2}')
 
 TMPDIR=$(mktemp -d -p /tmp)
```

Comment:
is this tempdir on the host? (it looks like it might be), if so can we clean it up in cluster-down perhaps?

Reply:
Any tmpdir will eventually be deleted by the host, and the artifacts aren't large. I do clean up any tmpdir that has large artifacts in it, but otherwise it might be sufficient to let the host clean it up.

Reply:
OK